tests/e2e/specs/BoundaryTimerEvent.spec.js

```diff
@@ -285,7 +285,36 @@ describe('Boundary Timer Event', () => {
 
   });
 
-  it('it cannot attach an incoming sequence flow on Boundary Timer Events');
+  it('it cannot attach an incoming sequence flow on Boundary Events', function() {
+    const taskForBoundaryEventsPosition = { x: 300, y: 200 };
+    dragFromSourceToDest(nodeTypes.task, taskForBoundaryEventsPosition);
+
+    const taskPosition = { x: 400, y: 300 };
+    dragFromSourceToDest(nodeTypes.task, taskPosition);
+
+    const boundaryEvents = [
+      nodeTypes.boundaryEvent,
+      nodeTypes.boundaryTimerEvent,
+      // @todo enable when added to menu
+      //nodeTypes.boundaryErrorEvent,
+      //nodeTypes.boundarySignalEvent,
```

Comment:
Can we remove these commented out lines? If we need them they'll still be in version control history. Right now they're just noise and are likely to just end up as cruft.

Reply:
If the commit is squashed, would there be a history? Should it be an issue to check these later?